zebrat: #namespace


    n_actions: 6 # We have 3 actions, Forwards,TurnLeft,TurnRight,Backwards
    # n_observations: 10 # We have 6 different observations

    speed_step: 1.0 # Time to wait in the reset phases

    linear_forward_speed: 0.6 # Spwwed for ging fowards
    linear_turn_speed: 0.6 # Lienare speed when turning
    angular_speed: 0.5 # Angular speed when turning Left or Right
    init_linear_forward_speed: 0.0 # Initial linear speed in which we start each episode
    init_linear_turn_speed: 0.0 # Initial angular speed in shich we start each episode
    
    new_ranges: 12 # How many laser readings we jump in each observation reading, the bigger the less laser resolution
    min_range: 0.31 # Minimum meters below wich we consider we have crashed
    max_laser_value: 6.0 # Value considered Ok, no wall
    min_laser_value: 0 # Value considered there is an obstacle or crashed
    
    desired_pose:
      x: 2.0
      y: 2.5
      z: 0.0
    
    forwards_reward: 5 # Points Given to go forwards
    turn_reward: 5 # Points Given to turn as action
    end_episode_points: 50 # Points given when ending an episode
    
    gridmap_size_x: 300
    gridmap_size_y: 300


    gridmap_resolution: 0.01 # 0.01m



    
    # environment_name = 'WamvNavTwoSetsBuoys-v0'
    

    
    
